### ISSUE TITLE
can't init with hubic error \x03

### ISSUE THREAD

**Issue by marcert** [gilbertchen/duplicacy#375]
Hi,
I want to test if the performance of hubic is good enough for my backups but if during initialization I get the following error message:
```
# duplicacy -d init ownCloud one://backup/
Reading the environment variable DUPLICACY_ONE_TOKEN
Failed to store the value to the keyring: keyring: No suitable keyring provider found (check your build flags)
Enter the path of the OneDrive token file (downloadable from https://duplicacy.com/one_start):/root
Failed to load the OneDrive storage at one://backup/: invalid character '\x03' looking for beginning of value

```
Unfortunately that is all I get.
Has somebodey an idea what happens ther?

Thanks and Kind Regards,
Marco

**Comment by TheBestPessimist** [comment]
Is `/root` there correct?
Do you have a file in `/` which is named `root` and contains the oneDrive token?

**Comment by marcert** [comment]
Yes I do- I mentioned in the title hubic- I tried it with hubic and onedrive and both deliver the same error message. If I change the directory to another one I get a different error message (...no such file or directory)

**Comment by gilbertchen** [comment]
It is strange that both hubic and onedrive gave the same error message.  Does `/root` look like a valid jason file?
